modyn/trainer_server/internal/trainer/pytorch_trainer.py

```diff
@@ -374,18 +375,18 @@ def train(self) -> None:  # pylint: disable=too-many-locals, too-many-branches
 
         total_stopw.stop("TotalTrain")
 
-        self._info(f"Finished training: {self._num_samples} samples, {batch_number + 1} batches.")
+        self._info(f"Finished training: {self._num_samples} samples, {passed_batches} batches.")
         self._log["num_samples"] = self._num_samples
         self._log["num_samples_trained"] = trained_batches * self._batch_size
-        self._log["num_batches"] = batch_number + 1
+        self._log["num_batches"] = passed_batches
```

Comment:
Should we log passed batches and trained batches separately?

Reply:
added